views/settings.html

```diff
@@ -290,6 +290,12 @@
     'testEmailBody': __('Yup, it works! You can start blasting away emails to the moon.'),
     'emailSent': __('The email has been sent! Check your inbox.'),
     'orCancel': __('or Cancel'),
+
+    'engagementTrackingTitle': __('Engagement analytics tracking'),
+    'engagementTrackingDescription': __('Choose the balance between subscriber privacy and deeper engagement insights that is right for your business.'),
+    'engagementTrackingFull': __('Full - additionally use browser cookies to improve the site and email engagement tracking and abandoned cart email accuracy.'),
```

Comment:
I think it would look nicer if you used a dash here instead of a hyphen -. You can use n-dash: – or m-dash: —